projects/packages/publicize/src/class-publicize-base.php

```diff
@@ -219,8 +219,12 @@ public function __construct() {
 		// Connection test callback.
 		add_action( 'wp_ajax_test_publicize_conns', array( $this, 'test_publicize_conns' ) );
 
-		add_action( 'init', array( $this, 'add_post_type_support' ) );
+		add_action( 'init', array( $this, 'add_post_type_support' ), 8 );
```

Comment:
Do we need the custom priority for this one too?

Reply:
So we call `get_post_types_by_support()` inside the action handler in order to decide which post types to add `post-thumbnails` support to. However `get_post_types_by_support( 'publicize' )` doesn't return anything until `add_post_type_support()` is called. Since the action handler below gets called in priority 8, this one needs to as well.

I should have left a comment. I'll try to think of something succinct that explains that.

A lot of the messiness in this PR could be avoided if we didn't worry about making an easy way to undo the behaviour of this PR. But I think it's worth doing because we are forcing a theme to support something when the theme developer may not expect that.